bundles-lib/pom.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software
+	Foundation (ASF) under one or more contributor license agreements. See the
+	NOTICE file distributed with this work for additional information regarding
+	copyright ownership. The ASF licenses this file to You under the Apache License,
+	Version 2.0 (the "License"); you may not use this file except in compliance
+	with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software distributed
+	under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+	OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+  the specific language governing permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.metron</groupId>
+        <artifactId>Metron</artifactId>
+        <version>0.4.0</version>
+    </parent>
+
+    <artifactId>bundles-lib</artifactId>
```

Comment:
metron-extensions/README.md has it spelled out.

Basically, an extension is a 'special' form of the existing metron assemblies, which package up the configuration and patterns and uber jars.
The difference ( besides parser relative pathing/target locations ) is that instead of an uber jar, an extension packages the executable code in a bundle.

So - the extension is the sum of its loadable code ( bundle ) and it's configurations/patterns.
At least for parsers.  Each extension type will be bundle + ???